src/images/XSVG.js

```diff
@@ -0,0 +1,19 @@
+// @flow
+
+import React from 'react';
+import styled from 'styled-components';
+
+import colors from '../styles/colors';
+
+const ExpiredLabel = styled.h5`
```

Comment:
Looks odd when alone, if we want to use a new text component, let's create it in the `atoms` folder and apply the same styling as other header text components

Reply:
e1454ea